LayoutKitTests/ReloadableViewLayoutAdapterCollectionViewTests.swift

```diff
@@ -64,75 +64,75 @@ private class TestCollectionView: LayoutAdapterCollectionView, TestableReloadabl
         reloadDataCount += 1
     }
 
-    private override func insertItemsAtIndexPaths(indexPaths: [NSIndexPath]) {
-        super.insertItemsAtIndexPaths(indexPaths)
-        batchUpdates.insertItems.appendContentsOf(indexPaths)
+    private override func insertItems(at indexPaths: [IndexPath]) {
+        super.insertItems(at: indexPaths)
+        batchUpdates.insertItems.append(contentsOf: indexPaths)
     }
 
-    private override func deleteItemsAtIndexPaths(indexPaths: [NSIndexPath]) {
-        super.deleteItemsAtIndexPaths(indexPaths)
-        batchUpdates.deleteItems.appendContentsOf(indexPaths)
+    private override func deleteItems(at indexPaths: [IndexPath]) {
+        super.deleteItems(at: indexPaths)
+        batchUpdates.deleteItems.append(contentsOf: indexPaths)
     }
 
-    private override func moveItemAtIndexPath(indexPath: NSIndexPath, toIndexPath newIndexPath: NSIndexPath) {
-        super.moveItemAtIndexPath(indexPath, toIndexPath: newIndexPath)
+    private override func moveItem(at indexPath: IndexPath, to newIndexPath: IndexPath) {
+        super.moveItem(at: indexPath, to: newIndexPath)
         batchUpdates.moveItems.append(ItemMove(from: indexPath, to: newIndexPath))
     }
 
-    private override func insertSections(sections: NSIndexSet) {
+    private override func insertSections(_ sections: IndexSet) {
         super.insertSections(sections)
-        batchUpdates.insertSections.addIndexes(sections)
+        batchUpdates.insertSections.formUnion(sections)
     }
 
-    private override func deleteSections(sections: NSIndexSet) {
+    private override func deleteSections(_ sections: IndexSet) {
         super.deleteSections(sections)
-        batchUpdates.deleteSections.addIndexes(sections)
+        batchUpdates.deleteSections.formUnion(sections)
     }
 
-    private override func moveSection(section: Int, toSection newSection: Int) {
+    private override func moveSection(_ section: Int, toSection newSection: Int) {
         super.moveSection(section, toSection: newSection)
         batchUpdates.moveSections.append(SectionMove(from: section, to: newSection))
     }
 
-    private func resetTestCounts() {
+    fileprivate func resetTestCounts() {
```

Comment:
some functions here have been changed to just `private` - can the other `fileprivate` functions be updated as well since some of them are already updated?